core/storage/statistics/gae_models.py

```diff
@@ -1285,7 +1285,7 @@ def get_model(
         return exploration_stats_model
 
     @classmethod
-    def create(
+    def create_and_put(
```

Comment:
I don't think you can change the semantics of this without changing what create() means for all other model classes as well.

Reply:
Thanks for pointing it out. I think I was a bit confused about what `create` is expected to do because I noticed that in some of the models (e.g. `ExplorationUserDataModel`, `GeneralFeedbackThreadModel`, `StoryProgressModel`) we use the create method to simply "create" the model instance but not put it.

Reply:
Ah ok, I didn't realize that (the examples I looked at went the other way, I think). I'm fine with anything as long as it results in a globally consistent approach.

Reply:
I think I would be fine for quickly fixing this issue, but I think we should add an issue for verifying that everything is unified.

Also should we call `create` inside this method?

Reply:
I've reverted this.